### PR TITLE
fix versions in `packages/tests`

### PR DESCRIPTION
**Pull request by KATT**
Closes #

## 🎯 Changes

fix versions in `packages/tests`

@juliusmarminge can you tell why the versions are not automatically bumped in the test? 🤔 

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/next/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: packages/tests/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tests",
+  "name": "@trpc/tests",
```
Comment (juliusmarminge) [inline packages/tests/package.json:2]: if we can get away with not naming this @trpc it makes our lives a lot easier.

all the install filters use @trpc/* and that would make this package match that too => unnecessary package installs in all examples and www

Comment (KATT) [reply packages/tests/package.json:2]: I'm unsure if this even helps tbh.... but won't turborepo just cache it anyway?

Comment (juliusmarminge) [reply packages/tests/package.json:2]: not the dependency installation. we can add another filter that removes this

```bash
pnpm --filter="@trpc/*" --filter="\!trpc@tests" install
```

Comment (KATT) [reply packages/tests/package.json:2]: Cool! Feel free to commit on this branch if you think this is the way :)

**Comment by juliusmarminge** [comment]
I've updated the Vercel setting to use this filter too.

**Pull request merged**